src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/DiscoveryAPI.feature.disabled

```diff
@@ -0,0 +1,50 @@
+Feature: Discovery API that retrieves the information from the base endpoint of the API
+
+        @ignore
+        Scenario: 01 Retrieves information of the API
+             Given a get to the root of the API
+             Then it should respond with 200
+              And the response body is
+                  """
+                  {
+                      "openapi": "3.1.0"
+                      "info": {
+                          "version": "1.0.0",
+                          "applicationName": "Ed-Fi Alliance Data Management Service"
+                      }
```

Comment:
These are not correct. `openapi` is not one of the attributes returned by Discovery API, and `version` and `applicationName` are not nested under `info`. I can see how it is a little confusing - you've mixed in a little bit of the Open API specification format that _describes_ the Discovery API.

I should have thought to tell you about [Swagger Editor](https://editor.swagger.io/). If you save the [specification yml file](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-API-Standards/blob/main/api-specifications/discovery/2.0-draft/discovery-api-2.0.yml) on your computer then you can load it into the Swagger Editor. Then you'll more easily see what the expected responses are.

![image](https://github.com/user-attachments/assets/1dc6ce3b-0b9f-4518-9094-1de388af1199)

Reply:
Done